### PR TITLE
[197] The Software Developer Conditionals - Mobile #181

### PR DESCRIPTION
**Pull request by rubyade**
Please find the first Java task for review

### CODE REVIEW

**Comment by Ireti203** [comment]
Add a screenshot of the output on your console.

**Comment by rubyade** [comment]
IMAG1645


> Add a screenshot of the output on your console.

A screenshot has been added. Many thanks.

**Comment by Ireti203** [comment]
Your screenshots are not usually clear. You can use Alt + PrtScn to take a screenshot if you are using Windows.

File: javaScript/conditionals.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Javascript Task</title>
+    <script src="conditionals.js"></script>
```
Comment (Ireti203) [inline javaScript/conditionals.html:6]: Your script should be placed after the closing of the body tag and not in the head tag.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: javaScript/conditionals.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    i++;
+    str +=i;
+}
+while (i < 116)
+
+console.log(str);
+for (var str = ""; str < 116; str+=i) {
+if (str / 3 === 0) { 
+    console.log("software");
+    } if (str / 5 === 0) {
```
Comment (Ireti203) [inline javaScript/conditionals.js:14]: Write the next statement on the next line. Also you dont have to keep repeating "if", you can use "else if"

File: javaScript/conditionals.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    console.log("developer");
+    } if (str / 3 === 0 && str / 5 === 0); {
```
Comment (Ireti203) [inline javaScript/conditionals.js:16]: Same here

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**